rim-page.html

```diff
@@ -0,0 +1,270 @@
+            <div class="row">
+                <div class="column">
+                    <h2>Market Leading Platform for Enabling IoT</h2>
+                    <p>BlackBerry provides organizations with software to secure the IoT devices they use, the products they make, the data they share and their communications.</p>
+                </div>
+            </div>                            
+            <div class="row">
+                <div class="column">
+                    <!-- img takes up the whole background of div -->
+                    <img src="people-laptop.jpg" alt="two people on laptop">
+                    <a href="#Management">Endpoint Management</a>
+                </div>
+                <div class="column">
+                    <!-- img takes up the whole back ground of div -->
+                    <img src="green-dot.jpg" alt="a wave of green dots">
+                    <a href="#Endpoint">Endpoint Security</a>    
+                </div>
+                <div class="column">
+                    <!-- img takes up the whole back ground of div -->
+                    <img src="men-cell-phone.jpg" alt="men holding cell phone">
+                    <a href="#Communication">Secure Communications</a>    
+                </div>
+                <div class="column">
+                    <!-- img takes up the whole back ground of div -->
+                    <img src="men-cell-phone-2.jpg" alt="men holding cell phone">
+                    <a href="#Alerts">Alerts and Crisis Communications</a>    
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <!-- img takes up the whole background of div -->
+                    <img src="computer-screen.jpg" alt="two people looking at one computer screen">
+                    <a href="#Documents">Secure Documents</a>
+                </div>
+                <div class="column">
+                    <!-- img takes up the whole back ground of div -->
+                    <img src="files.jpg" alt="a women looking at files">
+                    <a href="#Applications">Applications</a>    
+                </div>
+                <div class="column">
+                    <!-- img takes up the whole back ground of div -->
+                    <img src="computer-screen-showing.jpg" alt="a man looking at computer screen with part of car on screen">
+                    <a href="#Embedded">Embedded Systems</a>    
+                </div>
+                <div class="column">
+                    <!-- img takes up the whole back ground of div -->
+                    <img src="transport-truck.jpg" alt="image of transport truck">
+                    <a href="#">Transporting Asset Tracking</a>    
+                </div>
+            </div>
+            <!-- slideshow is treated as two static photos beside each other -->
+            <div class="row">
+                <div class="col-4">
```

Comment:
This is great.  As you can see, col-4 has 50% width, and you have two columns in this row, which makes up 100% of the width.  Most of the time, column widths have to add up to 100%.